tensorboard/webapp/runs/effects/runs_effects.ts

```diff
@@ -217,12 +218,18 @@ export class RunsEffects {
         }
         return {newRuns, runsForAllExperiments};
       }),
-      tap(({newRuns, runsForAllExperiments}) => {
+      withLatestFrom(this.store.select(getDashboardExperimentNames)),
+      map(([runsData, expNameByExpId]) => {
```

Comment:
This map() essentially just renames variables. Can we just include this logic in the tap() below?